[09/Dec/2014 19:32:10] "GET /monitor/ HTTP/1.1" 200 564636
[09/Dec/2014 19:32:30] "GET /monitor/linkstats/2716 HTTP/1.1" 200 61858
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False.
CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False.
CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False.
CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False.
CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False.
CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False.
CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False.
CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False.
Traceback (most recent call last):
  File "/home/m/MediaMonitor/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 345, in execute
    settings.INSTALLED_APPS
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 46, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 42, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 94, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/m/MediaMonitor/MediaMonitor/settings.py", line 32
    ALLOWED_HOSTS = [0.0.0.0]
                         ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/m/MediaMonitor/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 345, in execute
    settings.INSTALLED_APPS
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 46, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 42, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 94, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/m/MediaMonitor/MediaMonitor/settings.py", line 32
    ALLOWED_HOSTS = [0.0.0.0]
                         ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/m/MediaMonitor/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 345, in execute
    settings.INSTALLED_APPS
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 46, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 42, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 94, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/m/MediaMonitor/MediaMonitor/settings.py", line 32
    ALLOWED_HOSTS = [0.0.0.0]
                         ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/m/MediaMonitor/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 345, in execute
    settings.INSTALLED_APPS
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 46, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 42, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 94, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/m/MediaMonitor/MediaMonitor/settings.py", line 32
    ALLOWED_HOSTS = [0.0.0.0]
                         ^
SyntaxError: invalid syntax
[09/Dec/2014 19:59:21] "GET /monitor/ HTTP/1.1" 400 26
[09/Dec/2014 19:59:26] "GET /monitor/ HTTP/1.1" 400 26
[09/Dec/2014 19:59:38] "GET /monitor/linkstats/2716 HTTP/1.1" 400 26
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
[09/Dec/2014 20:00:55] "GET /monitor/linkstats/2716 HTTP/1.1" 200 61826
[09/Dec/2014 20:00:55] "GET /static/js/d3.min.js HTTP/1.1" 304 0
[09/Dec/2014 20:00:55] "GET /static/js/c3-0.3.0/c3.css HTTP/1.1" 304 0
[09/Dec/2014 20:00:55] "GET /static/js/c3-0.3.0/c3.min.js HTTP/1.1" 304 0
[09/Dec/2014 20:01:11] "GET /monitor/ HTTP/1.1" 200 564636
[09/Dec/2014 20:01:52] "GET /monitor/ HTTP/1.1" 200 564636
[09/Dec/2014 20:15:20] "GET /monitor/ HTTP/1.1" 200 564636
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
[09/Dec/2014 20:23:45] "GET /monitor/ HTTP/1.1" 200 564636
[09/Dec/2014 20:31:50] "GET /monitor/ HTTP/1.1" 200 584525
[09/Dec/2014 20:34:04] "GET /monitor/ HTTP/1.1" 200 590359
[09/Dec/2014 20:35:56] "GET /monitor/ HTTP/1.1" 200 593736
[09/Dec/2014 20:36:22] "GET /monitor/ HTTP/1.1" 200 593737
[09/Dec/2014 20:37:13] "GET /monitor/ HTTP/1.1" 200 600372
[09/Dec/2014 20:40:59] "GET /monitor/ HTTP/1.1" 200 662401
[09/Dec/2014 20:41:30] "GET /monitor/linkstats/6939 HTTP/1.1" 200 4728
[09/Dec/2014 21:01:51] "GET /monitor/ HTTP/1.1" 200 930488
[09/Dec/2014 21:08:15] "GET /monitor/ HTTP/1.1" 200 1105265
[09/Dec/2014 21:57:58] "GET /monitor/ HTTP/1.1" 200 1840509
[09/Dec/2014 22:13:57] "GET /monitor/ HTTP/1.1" 200 1840509
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
ddddd

Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
Error: That port is already in use.
[15/Dec/2014 16:35:11] "GET /monitor/ HTTP/1.1" 200 3358258
[15/Dec/2014 16:35:22] "GET /favicon.ico HTTP/1.1" 404 85
[20/Dec/2014 20:24:06] "GET /monitor/ HTTP/1.1" 200 4735589
[20/Dec/2014 20:24:16] "GET /favicon.ico HTTP/1.1" 404 85
[20/Dec/2014 20:25:05] "GET /monitor/linkstats/16635 HTTP/1.1" 200 2846
[20/Dec/2014 20:25:06] "GET /static/js/d3.min.js HTTP/1.1" 404 93
[20/Dec/2014 20:35:30] "GET /monitor/linkstats/16636 HTTP/1.1" 200 2851
[20/Dec/2014 20:35:30] "GET /static/js/d3.min.js HTTP/1.1" 404 93
[20/Dec/2014 20:36:39] "GET /static/js/d3.js HTTP/1.1" 404 89
[20/Dec/2014 20:36:47] "GET /monitor/static/js/d3.js HTTP/1.1" 404 97
Traceback (most recent call last):
  File "/usr/lib/python2.7/wsgiref/handlers.py", line 86, in run
    self.finish_response()
  File "/usr/lib/python2.7/wsgiref/handlers.py", line 127, in finish_response
    self.write(data)
  File "/usr/lib/python2.7/wsgiref/handlers.py", line 215, in write
    self._write(data)
  File "/usr/lib/python2.7/socket.py", line 324, in write
    self.flush()
  File "/usr/lib/python2.7/socket.py", line 303, in flush
    self._sock.sendall(view[write_offset:write_offset+buffer_size])
error: [Errno 104] Connection reset by peer
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 593, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/local/lib/python2.7/dist-packages/django/core/servers/basehttp.py", line 129, in __init__
    super(WSGIRequestHandler, self).__init__(*args, **kwargs)
  File "/usr/lib/python2.7/SocketServer.py", line 651, in __init__
    self.finish()
  File "/usr/lib/python2.7/SocketServer.py", line 704, in finish
    self.wfile.flush()
  File "/usr/lib/python2.7/socket.py", line 303, in flush
    self._sock.sendall(view[write_offset:write_offset+buffer_size])
error: [Errno 32] Broken pipe
[20/Dec/2014 20:41:20] "GET /monitor/ HTTP/1.1" 200 4735589
Unhandled exception in thread started by <function wrapper at 0x3b8d1b8>[20/Dec/2014 20:41:30] "GET /favicon.ico HTTP/1.1" 404 1938
[20/Dec/2014 20:41:30] "GET /favicon.ico HTTP/1.1" 404 1938
[20/Dec/2014 20:47:47] "GET /monitor/static/js/d3.js HTTP/1.1" 404 2245
[20/Dec/2014 20:48:12] "GET /static/js/d3.js HTTP/1.1" 200 325951
[20/Dec/2014 20:48:21] "GET /static/js/d3.min.js HTTP/1.1" 200 146528
[20/Dec/2014 20:48:33] "GET /static/js/d3.min.js HTTP/1.1" 404 93
[22/Dec/2014 01:16:11] "GET /monitor/ HTTP/1.1" 500 27
[22/Dec/2014 01:16:11] "GET /favicon.ico HTTP/1.1" 404 85
[22/Dec/2014 01:16:11] "GET /favicon.ico HTTP/1.1" 404 85
Unhandled exception in thread started by <function wrapper at 0x30031b8>
Error in sys.excepthook:
Traceback (most recent call last):
[22/Dec/2014 01:16:38] "GET /monitor/ HTTP/1.1" 500 53155
[22/Dec/2014 01:17:12] "GET /monitor/ HTTP/1.1" 200 27373
[22/Dec/2014 01:19:09] "GET /monitor/ HTTP/1.1" 200 26357
[22/Dec/2014 01:20:18] "GET /monitor/ HTTP/1.1" 200 26332
[22/Dec/2014 01:23:41] "GET /monitor/ HTTP/1.1" 200 26393
[22/Dec/2014 01:25:39] "GET /monitor/ HTTP/1.1" 200 26431
[22/Dec/2014 01:25:57] "GET /monitor/ HTTP/1.1" 200 26432
[22/Dec/2014 01:27:16] "GET /monitor/ HTTP/1.1" 200 26510
[22/Dec/2014 01:27:26] "GET /monitor/ HTTP/1.1" 200 26510
[22/Dec/2014 01:28:39] "GET /monitor/linkstats/16663 HTTP/1.1" 200 2849
[22/Dec/2014 01:28:40] "GET /static/js/c3-0.3.0/c3.min.js HTTP/1.1" 304 0
[22/Dec/2014 01:28:40] "GET /static/js/c3-0.3.0/c3.css HTTP/1.1" 304 0
[22/Dec/2014 01:28:40] "GET /static/js/d3.min.js HTTP/1.1" 200 146528
[22/Dec/2014 02:29:09] "GET /monitor/ HTTP/1.1" 200 26510
[22/Dec/2014 02:29:15] "GET /monitor/ HTTP/1.1" 200 26510
[22/Dec/2014 02:29:23] "GET /monitor/?page=2 HTTP/1.1" 200 27453
[22/Dec/2014 02:29:27] "GET /monitor/?page=3 HTTP/1.1" 200 27068
[22/Dec/2014 02:29:31] "GET /monitor/?page=10 HTTP/1.1" 200 27194
[22/Dec/2014 02:29:35] "GET /monitor/?page=100 HTTP/1.1" 200 27836
[22/Dec/2014 02:29:54] "GET /monitor/?page=194 HTTP/1.1" 200 10062
[22/Dec/2014 02:33:04] "GET /monitor/?page=21 HTTP/1.1" 200 26478
[22/Dec/2014 02:33:12] "GET /monitor/?page=1 HTTP/1.1" 200 26510
[22/Dec/2014 02:46:02] "GET /monitor/?page=1 HTTP/1.1" 200 26510
[22/Dec/2014 02:46:23] "GET /monitor/linkstats/16661 HTTP/1.1" 200 2844
[22/Dec/2014 10:45:55] "GET / HTTP/1.1" 404 1905
[22/Dec/2014 10:45:59] "GET /monitor/ HTTP/1.1" 200 26510
[22/Dec/2014 17:27:59] "GET /monitor/ HTTP/1.1" 200 26177
[24/Dec/2014 00:48:47] "GET /monitor/ HTTP/1.1" 200 26225
[24/Dec/2014 02:45:53] "GET /monitor/ HTTP/1.1" 200 26225
[29/Dec/2014 19:05:26] "GET /monitor/ HTTP/1.1" 200 26225
[29/Dec/2014 19:05:27] "GET /favicon.ico HTTP/1.1" 404 1938
[29/Dec/2014 19:05:27] "GET /favicon.ico HTTP/1.1" 404 1938
